cypress/e2e/gui/home-page-gui.cy.js

```diff
@@ -0,0 +1,45 @@
+/// <reference types="Cypress" />
+
+describe("EngageSphere", () => {
+  beforeEach(() => {
+    cy.visit("https://engage-sphere.vercel.app/");
+  });
+
+  it("Renderiza o cabeçalho com um h1 e alternância de tema", () => {
+    cy.get("h1").contains("EngageSphere");
```

Comment:
Que tal assim?

```suggestion
    cy.contains("h1", "EngageSphere").should("be.visible");
```

Lembre da verificação de que o elemento está visível.